src/main/react/features/login/LoginFeature.tsx

```diff
@@ -0,0 +1,105 @@
+import {Box, Card, CardContent, CardHeader, Container, Grid, TextField} from "@material-ui/core";
+import React, {Fragment} from "react";
+import {useLoginType} from "../../hooks/LoginTypeHook";
+import Button from "@material-ui/core/Button";
+import {makeStyles} from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import CircularProgress from "@material-ui/core/CircularProgress";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+    minHeight: '100vh',
+    backgroundImage: "linear-gradient(to bottom, white," + theme.palette.primary.main + ")",
+  },
+  background: {
+    width: '100%',
+    minHeight: '100vh',
+    backgroundImage: "url('/images/background15.webp')",
+    backgroundSize: "80%",
+    backgroundRepeat: "no-repeat",
+    backgroundPosition: "center",
+  },
+  loginContent: {
+    display: 'flex',
+    alignItems: "center",
+    flexDirection: 'column',
+  },
+  card: {
+    minWidth: "20rem",
+    maxWidth: "30rem",
+    marginLeft: "1rem",
+    marginRight: "1rem",
+  },
+  cardHeader: {
+    backgroundColor: theme.palette.primary.main,
+  },
+  cardContent: {
+    display:"flex",
+    flexDirection: "column",
+    alignItems: "center"
+  },
+  formLogin: {
+    width: "100%",
+    display:"flex",
+    flexDirection: "column",
+    alignItems: "stretch",
+    gap: "2rem"
+  },
+  logo: {
+    marginTop: "4rem",
+    marginBottom: "2rem",
+    maxWidth: "600px",
+  },
+  flock: {
+    fontFamily: "monospace",
+    marginTop: "1rem"
+  },
+}))
+
+export function LoginFeature() {
+
+  const classes = useStyles();
+  const loginType = useLoginType();
+  const isLoading = loginType.type == "LOADING";
+  const isGoogle = loginType.type == "GOOGLE";
+
+  const googleIconLogo = <img width="46px" height="46px" src="images/googleLogoIcon.svg" alt="Google Logo Icon"/>
+
+  return (
+    <Box className={classes.root}>
+      <Box className={classes.background}>
+        <Box className={classes.loginContent}>
+          <Box className={classes.logo}>
+            <img width="100%" src="images/workday_logo.webp"/>
+          </Box>
+          <Card className={classes.card}>
+            <CardHeader className={ classes.cardHeader } title="Workday Login"></CardHeader>
```

Comment:
This title becomes a span so it's better to check what kind of heading this should be (probably h1) hand then use the `Typography` component as a title